### ISSUE TITLE
Task 7 rejected

### ISSUE THREAD

**Issue by Kuzirashi** [gabrycina/crud-dapp#1]
This application is broken. It's impossible to create user etc. When you click submit page refreshes.

**Comment by gabrycina** [comment]
I’m really sorry. I thought it was working when I committed. Maybe I broke
it.

Can I fix it and re-submit?

**Comment by Kuzirashi** [comment]
Yes you can. It will be judged at the end after previous submissions are checked.
